scipy/spatial/qhull.pyx

```diff
@@ -541,6 +542,7 @@ cdef class _Qhull:
         cdef int i, j, ipoint, ipoint2, ncoplanar
         cdef object tmp
         cdef np.ndarray[np.npy_int, ndim=2] facets
+        cdef np.ndarray[np.npy_int, ndim=1] good
```

Comment:
Probably ok for now -- we're trying to encourage new Cython to use [typed memoryviews](https://cython.readthedocs.io/en/latest/src/userguide/memoryviews.html), but the rest of the code here is using the original approach, so that's maybe fine for now.

Reply:
I am always interested in learning, but I don't know much about Cython.